source/updating-website.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Updating the website
+---
+
+# Updating the website
+{:.no_toc}
+
+* Will be replaced with the ToC, excluding a header
+{:toc}
+
+## The setup
+
+The whole website is available in a dedicated Git [repository](https://gitbox.apache.org/repos/asf?p=struts-site.git), 
+you can also use a Github [mirror](https://github.com/apache/struts-site).
+
+The `master` branch contains the current source code of the website, any changes should be applied to this branch, 
+also any Pull Request should be opened against this branch as well.
+
+We are using Jekyll and mix of Markdown pages with raw Html pages, where Markdown is a preferred option.
+
+> WARN: do not manually change the `asf-site` or `ast-staging` branches, they will be automatically updated
+> by CI server. 
+
+## Applying a change
+
+If you are a contributor, and the change is small you can push it directly to the `master` branch. In any other case
+please open a Pull Request. The Pull Request will be automatically build and deployed to the [staging site](https://struts.staged.apache.org/).
+
```

Comment:
Shouldn't we refer to http://s.apache.org/asfyaml and BuildBot to help understand how the actual build is performed?

Reply:
It is, in the `The setup` section

Reply:
Ah, thanks - oversaw this!